cli.js

```diff
@@ -41,13 +41,13 @@ var cli = meow([
 	'  ava [<file|folder|glob> ...]',
 	'',
 	'Options',
-	'  --init       Add AVA to your project',
-	'  --fail-fast  Stop after first test failure',
-	'  --serial     Run tests serially',
-	'  --require    Module to preload (Can be repeated)',
-	'  --tap        Generate TAP output',
-	'  --verbose    Enable verbose output',
-	'  --no-cache   Disable the transpiler cache',
+	'  --init           Add AVA to your project',
+	'  --fail-fast      Stop after first test failure',
+	'  -s, --serial     Run tests serially',
```

Comment:
@jamestalmage That's only true when all flags have shortflags, which isn't the case here or pretty much ever.